### ISSUE TITLE
Hindi Language Support

### ISSUE THREAD

**Issue by hashd** [savoirfairelinux/num2words#226]
### Expected Behaviour

num2words should be able to convert numbers to words in Hindi (Indian Language).

### Actual Behaviour

Hindi Language support is currently not present

### Steps to reproduce
Not applicable

We've implemented Hindi language support for `num2words` to help us in our development. We can contribute it back if it falls within the scope.

**Comment by maximest-pierre** [comment]
Yes, it's within the scope of ```num2words``` as a matter of facts any new language or dialect is welcome. Just send a PR and we will look into adding it.

**Comment by hashd** [comment]
Thanks! At the moment, we have added Hindi language primarily because we needed it for our work. We'll create the PR corresponding to that soon.

We'll gladly contribute other languages/dialects when we add any for our use.

**Comment by erozqba** [comment]
Great news @hashd! Thanks a lot in advance for taking the time to make that contribution :smile:

**Comment by jqueguiner** [comment]
For anyone landing here: Hindi is supported in [num2words2](https://github.com/jqueguiner/num2words2), an actively-maintained fork.

```python
>>> from num2words2 import num2words
>>> num2words(5, lang='hi')
'पाँच'
```

Install: `pip install num2words2`.

---
*Status update: still resolved in num2words2 **v1.0.9** (latest release on PyPI / AUR `python-num2words2`, 2026-05-01). See https://github.com/jqueguiner/num2words2/releases for the full changelog since the fork.*
